js/jquery.imageplayer.js

```diff
@@ -257,7 +257,7 @@ if (typeof(jQuery) == 'undefined') alert('jQuery library was not found.');
             clearTimeout(rotator);
             i = images.length - 1;
             handle = document.getElementById('range');
-            handle.value = handle.max  
+            handle.value = handle.max;  
```

Comment:
Am I correct if I say you can use `images.length - 1` instead of reading the max value directly?

So, lines 259 and 260 would become: `scrubber.val(images.length - 1);`